chronosphere/tfschema/notification_policy.go

```diff
@@ -85,6 +85,11 @@ var NotificationRouteSchema = typeset.Set{
 		"repeat_interval": Duration{
 			Optional: true,
 		},
+		"group_by": typeset.NotNormalized(&schema.Schema{
```

Comment:
i'd recommend against using a set since we've seen a lot of issues with them, e.g., https://github.com/hashicorp/terraform-plugin-sdk/issues/652

Set also implies that the order is not important, but the API retains the order. Can we use a list here?